android/build.gradle

```diff
@@ -3,55 +3,30 @@
 buildscript {
     ext {
         firebaseMessagingVersion = "23.2.1"
-        kotlin_version = "1.7.0"
         buildToolsVersion = "33.0.2"
+        kotlinVersion = "1.7.0"
         minSdkVersion = 23
         compileSdkVersion = 33
         targetSdkVersion = 33
         supportLibVersion = "28.0.0"
-        
-        if (System.properties['os.arch'] == "aarch64") {
-            // For M1 Users we need to use the NDK 24 which added support for aarch64
-            ndkVersion = "24.0.8215888"
-        } else {
-            // Otherwise we default to the side-by-side NDK version from AGP.
-            ndkVersion = "21.4.7075529"
-        }
+        // We use NDK 24 which has both M1 support and is the side-by-side NDK version from AGP.
+        ndkVersion = "24.0.8215888"
     }
     repositories {
         google()
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.2.1'
+        classpath 'com.android.tools.build:gradle'
         classpath 'com.facebook.react:react-native-gradle-plugin'
-        classpath 'de.undercouch:gradle-download-task:5.0.1'
         classpath 'com.google.gms:google-services:4.3.3'
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
 
-        // NOTE: Do not place your application dependencies here; they belong
-        // in the individual module build.gradle files
     }
 }
 
 allprojects {
     repositories {
-        maven {
-            // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
-            url("$rootDir/../node_modules/react-native/android")
-        }
-        maven {
-            // Android JSC is installed from npm
-            url("$rootDir/../node_modules/jsc-android/dist")
-        }
-        mavenCentral {
-            // We don't want to fetch react-native from Maven Central as there are
-            // older versions over there.
-            content {
-                excludeGroup "com.facebook.react"
-            }
-        }
-        google()
```

Comment:
Why do we remove `google()` too?

Reply:
following the migration guide it is suggested to remove https://react-native-community.github.io/upgrade-helper/?from=0.70.15&to=0.72.15#RnDiffApp-android-build.gradle